pkg/apis/policy/v1alpha1/clusterimagepolicy_validation.go

```diff
@@ -218,6 +222,20 @@ func (source *Source) Validate(ctx context.Context) *apis.FieldError {
 	var errs *apis.FieldError
 	if source.OCI == "" {
 		errs = errs.Also(apis.ErrMissingField("oci"))
+	} else {
```

Comment:
WDYT about the suggestion here: https://github.com/google/go-containerregistry/pull/1474#issuecomment-1289024708 

This offloads more string munging into go-containerregistry which makes it their (er, my) responsibility to test 😄

Reply:
It's an option. But I'd avoid appending any fake value to what we get from the users. 
We still offload all the validations while using generic calls to count the parts.